contracts/MultiSigWallet.sol

```diff
@@ -21,63 +21,71 @@ pragma solidity ^0.4.23;
 //
 // ----------------------------------------------------------------------------
 
+import "./SafeMath.sol";
+
 
 /**
  * @title MultiSigWallet Contract
  *
- * @notice Implement operations which require multiple confirmations.
- *         It is inherited by TokenHolder.sol.
- *
+ * @notice Implement operations that require multiple confirmations.
  */
 contract MultiSigWallet {
 
+    /* Usings */
+
+    using SafeMath for uint256;
+
+
     /* Events */
 
-    event Proposed(
+    event Proposed (
         bytes32 indexed _transactionId,
         address indexed _sender
     );
 
-    event Confirmed(
+    event Confirmed (
         bytes32 indexed _transactionId,
         address indexed _sender
     );
 
-    event Revoked(
+    event Revoked (
         bytes32 indexed _transactionId,
         address indexed _sender
     );
 
-    event WalletReplaced(
+    event WalletReplaced (
         address indexed _sender,
         address indexed _oldWallet,
         address indexed _newWallet
     );
 
-    event WalletAdded(
+    event WalletAdded (
         bytes32 indexed _transactionId,
         address indexed _wallet
     );
 
-    event WalletRemoved(
+    event WalletRemoved (
         bytes32 indexed _transactionId,
         address indexed _wallet
     );
 
-    event RequirementChanged(
-        uint8 _required
+    event RequirementChanged (
+        uint256 _required
     );
 
-    /* Struct */
+
+    /* Structs */
 
     /**
-       It denotes which wallet has confirmed and status for the transaction.
-       Status values could be :-
-       0 :- initial state/Not proposed.
-       1 :- Proposed state.
-       2 :- Successfully executed state.
+     * Specifies which wallet has confirmed the proposed transaction and
+     * confirmation status for the transaction.
+     *
+     * Confirmation status values are:
+     *      0 : Initial (not proposed) state.
+     *      1 : Proposed state.
+     *      2 : Executed state.
      */
-    struct Confirmation{
+    struct Confirmation {
```

Comment:
Shouldn't this struct really be, "Transaction" (and other names changed accordingly)? Because confirmations aren't not proposed, proposed, and then executed, transactions are.

This change would also, I think, be more consistent with the `confirmations` mapping, which has transaction ID as the key, and functions named in the style of `isTransaction___`.

Reply:
+1 for first point.
Didn't completely understand isTransaction__ point.Lets discuss this on call.